src/utils/graphql/queries/getReview.query.ts

```diff
@@ -5,7 +5,8 @@ export default gql`
     review(id: $reviewId) {
       reviewer {
         id
-        username
```

Comment:
Why not leave `username` in? There might not be an immediate use, but can't hurt to have it available, right?